Formula/b/breseq.rb

```diff
@@ -0,0 +1,42 @@
+class Breseq < Formula
+  desc "Computational pipeline for finding mutations in short-read DNA resequencing data"
+  homepage "https://barricklab.org/breseq"
+  url "https://github.com/barricklab/breseq/archive/refs/tags/v0.39.0.tar.gz"
+  sha256 "5aa1bd9af71899e1358cfb9b8440c16cc908f185d9178a401a5a4d3f0c7ee861"
+  license all_of: ["GPL-2.0-or-later", "MIT", "BSD-3-Clause"]
```

Comment:
any source for this license composition?

Reply:
> LICENSE is GPL-2.0
> COPYING with GPL-3.0

that might be an issue (we should alert them about it)

> However, it looks like the repo is GPL-1.0-or-later from the [header comments](https://github.com/barricklab/breseq/blob/v0.39.0/src/c/breseq/alignment.cpp#L13-L15).


yeah, I found the same, I think that might be they forgot to update the headers when updating the license files.

Reply:
Will do. When running the `breseq` command, it outputs:

```text
breseq is free software; you can redistribute it and/or modify it under the
terms the GNU General Public License as published by the Free Software 
Foundation; either version 2, or (at your option) any later version.
```

Should I add comments about the license locations like the `redis` formula?

```ruby
  license all_of: [
    "GPL-2.0-or-later",
    "MIT", # extern/samtools-1.3.1, extern/samtools-1.3.1/htslib-1.3.1
    "BSD-3-Clause", # extern/samtools-1.3.1/htslib-1.3.1/cram
  ]
```

Reply:
sounds good 👍